solutions/search/get-started/elasticsearch-basics-quickstart.md

```diff
@@ -43,30 +49,11 @@ The following response indicates the index was created successfully.
 }
 ```
 
+:::
 ::::
+::::{step} Add a single document
 
-
-
-## Step 2: Add data to your index [getting-started-add-documents]
-
-::::{tip}
-This tutorial uses {{es}} APIs, but there are many other ways to [add data to {{es}}](ingest-for-search.md).
-
-::::
-
-
-You add data to {{es}} as JSON objects called documents. {{es}} stores these documents in searchable indices.
-
-
-### Add a single document [getting-started-add-single-document]
-
-Submit the following indexing request to add a single document to the `books` index.
-
-::::{tip}
-If the index didn’t already exist, this request would automatically create it.
-
-::::
-
+Submit the following indexing request to add a single document to the `books` index:
```

Comment:
Submit is the right verb here? I see it used a couple of times in this file. That led me to search through our other docs, and it looks like it's used quite a bit in our ES content. TIL.